rdtools/normalization.py

```diff
@@ -488,7 +488,11 @@ def energy_from_power(power, target_frequency=None, max_timedelta=None):
                          'DatetimeIndex')
 
     t_steps = t_step_nanoseconds(power)
-    median_step_ns = t_steps.median()
+    if power.index.freq is not None:
+        timedelta = pd.to_timedelta(power.index.freq)
+        median_step_ns = timedelta.total_seconds() * 1e9
```

Comment:
I did this before I saw the `pd.tseries.frequencies.to_offset(freq).nanos` approach below.  Is there reason to prefer one over the other?

Reply:
I don't know of a reason to prefer one over the other. The only thing to consider would be consolidating the repeated logic in a function `_freq_to_nanoseconds()` or similar. Borderline case in my opinion.